e2e/multi-svc-app/front-end/main.go

```diff
@@ -128,7 +130,30 @@ func PutEFSCheck(w http.ResponseWriter, req *http.Request, ps httprouter.Params)
 	w.WriteHeader(http.StatusOK)
 }
 
-func main() {
+func handleUDPTraffic(wg *sync.WaitGroup) {
+	defer wg.Done()
+
+	udpServer, err := net.ListenUDP("udp", &net.UDPAddr{Port: 8080})
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.Println("Listening on port 8080...")
+
+	for {
+		buffer := make([]byte, 1024)
```

Comment:
I feel like we shouldn't init this read buf within each loop. Instead, we should probably only init a print buf when the read succeeds and clear the read buf.